src/Eleventy.js

```diff
@@ -794,12 +793,10 @@ Arguments:
 		if (!activeQueue.length) {
 			return false;
 		}
-
-		return this.shouldTriggerConfigReset(
-			activeQueue.map((path) => {
-				return PathNormalizer.normalizeSeperator(TemplatePath.addLeadingDotSlash(path));
-			}),
+		activeQueue = new Set(
+			activeQueue.map(TemplatePath.addLeadingDotSlash).map(PathNormalizer.normalizeSeperator),
```

Comment:
I know it looks nice but wouldn't it be better to only map over it once?

Reply:
Well after prettier reformatted what I wrote (grumble), I'm not even sure it looks better. I'll change back.